apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -26,12 +26,38 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define	STACK_NORMAL_SIZE		10
+#define	STACK_NORMAL_SIZE		300
```

Comment:
What is the basis for deciding these values? Is it related to some default stack size? Can we comment about it here?

Reply:
Its just predefined size used for test